libs/sdk-js/index.d.cts

```diff
@@ -0,0 +1 @@
+export * from './dist/index.cjs'
```

Comment:
noob question - aren't these generated by `yarn build`? Or are we intending to check them in?

Reply:
actually great catch, i accidentally checked them in, let me remove